R/drive_mkdir.R

```diff
@@ -0,0 +1,83 @@
+#' Make a Google Drive Directory
+#'
+#' @param dir character, name of the folder you would like to create
+#' @param path character, path where you would like the folder on your Google Drive
+#'
+#' @return object of class `gfile` and `list` that contains uploaded folder's
+#'   information
+#' @export
+drive_mkdir <- function(dir = NULL, path = NULL) {
```

Comment:
To consider: should this be named -- or at least aliased -- to `drive_dir_create()` because `dir.create()`?